setup-pg_hba.sh

```diff
@@ -2,16 +2,40 @@
 
 source /env-data.sh
 
+SETUP_LOCKFILE="${ROOT_CONF}/.pg_hba.conf.lock"
+if [ -f "${SETUP_LOCKFILE}" ]; then
+	return 0
+fi
+
 # This script will setup pg_hba.conf
 
-# Custom IP range via docker run -e (https://docs.docker.com/engine/reference/run/#env-environment-variables)
-# Usage is: docker run [...] -e ALLOW_IP_RANGE='192.168.0.0/16'
+# Reconfigure pg_hba if environment settings changed
+# cat ${ROOT_CONF}/pg_hba.conf.template > ${ROOT_CONF}/pg_hba.conf
+
+
+if [[ "${FORCE_SSL}" =~ [Ff][Aa][Ll][Ss][Ee] ]]; then
+  PG_CONF_HOST='host'
+  CERT_AUTH=${PASSWORD_AUTHENTICATION}
```

Comment:
Is the PASSWORD_AUTH defined anywhere in the env-data

Reply:
Not that I can find. I shall copy and test again

Reply:
LOG:  invalid authentication method "scram-sha-256"

Reply:
Switched to md5 now it is running

Reply:
scram-sha-256 is only available in PostgreSQL 10 and above